### PR TITLE
feat: add autovac worker metrics to sql_exporter and grafana

### LINKED ISSUE #106: Add vacuum analysis queries & dashboard
Add a new query file for doing more in-depth vacuum analysis. 

See this blog post for details - http://dtrace.org/blogs/dap/2019/05/22/visualizing-postgresql-vacuum-progress/

Keep this as a separate file to allow it to be optional since it could potentially be very noisy. Recommend having monitoring query only target specific tables that can be set by user.

### PR DESCRIPTION
**Pull request by keithf4**
# Description  

Add metrics to track autovac workers running vs max.

Original issue had reference to more extensive vacuum metrics and dashboard, but it's been taken down. It would've been optional anyway

Please indicate what kind of change your PR includes (multiple selections are acceptable):

- [ ] Bugfix
- [x] Enhancement
- [ ] Breaking Change
- [ ] Documentation

PRs should be against existing issues, so please list each issue using a separate 'closes' line:

closes #106 

If this PR depends on another PR or resolution of another issue, please indicate that here using a separate 'depends' line for each dependency.

depends on #

If you have an **external** dependency (packages, portal updates, etc), add the 'BLOCKED' tag to your PR.

## Testing
*None of the testing listed below is optional.*

- Installation method:  
    - [ ] Binary install from source, version:  
    - [x] OS package repository, distro, and version:  
    - [ ] Local package server, version:  
    - [ ] Custom-built package, version:  
    - [ ] Other:  
- [x] PostgreSQL, Specify version(s):  
- [ ] docs tested with hugo version(s):  

### Code testing

Have you tested your changes against:
- [x] RedHat/CentOS
- [ ] Ubuntu
- [ ] SLES
- [ ] Not applicable

If your code touches postgres_exporter, have you:
- [x] Tested against all versions of PostgreSQL affected
- [x] Ensure that exporter runs with no scrape errors
- [ ] Not applicable

If your code touches node_exporter, have you:
- [ ] Ensure that exporter runs with no scrape errors
- [ ] Not applicable

If your code touches Prometheus, have you:
- [ ] Ensured all configuration changes pass `promtool check config`
- [ ] Ensured all alert rule changes pass `promtool check rules`
- [ ] Prometheus runs without issue
- [ ] Alertmanager runs without issue
- [ ] Not applicable

If your code touches Grafana, have you:
- [ ] Ensured Grafana runs without issue
- [ ] Ensured relevant dashboards load without issue
- [ ] Not applicable

### Checklist:
- I have made corresponding changes to:  
    - [ ] the documentation  
    - [ ] the release notes  
    - [ ] the upgrade doc

### CODE REVIEW

**Review by hunleyd — DISMISSED** [review]

**Review by jchancojr — DISMISSED** [review]

**Review by YorviArias — DISMISSED** [review]

**Review by hunleyd — APPROVED** [review]

**Review by hunleyd — CHANGES_REQUESTED** [review]
conflicts

**Review by jchancojr — CHANGES_REQUESTED** [review]
pending conflicts

**Review by YorviArias — CHANGES_REQUESTED** [review]
still has conflicts

File: grafana/postgres/PG_Details.json [hunk 1]
```diff
@@ -2346,7 +2454,7 @@
         },
         "datasource": {
           "type": "prometheus",
-          "uid": "${ccp_datasource}"
+          "uid": "PDC1078F23EBDF0E5"
```
Comment (hunleyd) [inline grafana/postgres/PG_Details.json:2457]: i don't know enough about dashboard. this changed from a var to a constant? that's intended/correct?

**Review by hunleyd — CHANGES_REQUESTED** [review]

File: grafana/postgres/PG_Details.json (hunk 1 above)
Comment (hunleyd) [reply grafana/postgres/PG_Details.json:2457]: err, about dashboards. like, in general

Comment (keithf4) [reply grafana/postgres/PG_Details.json:2457]: Yeah when I did the upstream merge of this it messed it up and undid the change to a variable. It was easier to just refix the dashboard like I did for the variable change PR, but there's still a few spots that the old value is there that I don't know how to fix properly. It's mostly around the default values for the dropdowns at the time the export is done. It seems to be working, but I'll check things again after this PR is merged.

**Review by hunleyd — APPROVED** [review]

**Review by YorviArias — APPROVED** [review]

**Review by jchancojr — APPROVED** [review]

**Pull request merged**